scripts/generate_remote_daemon_release_manifest.py

```
@@ -0,0 +1,66 @@
+import sys
+import urllib.parse
+from pathlib import Path
+
+
+def main() -> None:
+    (
+        version,
+        release_tag,
+        repo,
+        checksums_asset_name,
+        checksums_path,
```

Ruff (0.15.21)

[warning] Unpacked variable `checksums_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)